client/src/components/TestSuite/TestSuiteDetails/TestListItem/InputOutputsList.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC } from 'react';
+import useStyles from './styles';
+import {
+  Table,
+  TableBody,
+  TableRow,
+  TableCell,
+  Typography,
+  TableHead,
+  ListItem,
+} from '@mui/material';
+import { TestInput, TestOutput } from 'models/testSuiteModels';
+import ReactMarkdown from 'react-markdown';
+
+interface InputOutputsListProps {
+  inputOutputs: TestInput[] | TestOutput[];
+  noValuesMessage?: string;
+  headerName: string;
+}
+
+const InputsOutputsList: FC<InputOutputsListProps> = ({
+  inputOutputs,
+  noValuesMessage,
+  headerName,
+}) => {
+  const styles = useStyles();
+
+  const headerTitles = [headerName, 'Value'];
+  const inputOutputsListHeader = (
+    <TableRow key="msg-header">
+      {headerTitles.map((title) => (
+        <TableCell key={title} className={title === 'Value' ? styles.messageMessage : ''}>
```

Comment:
Should probably rename these styles at some point. Doesn't have to be here.

Reply:
Updated style name.